paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormViewModel.kt

```diff
@@ -42,12 +48,32 @@ internal class FormViewModel @Inject internal constructor(
         @Suppress("UNCHECKED_CAST")
         override fun <T : ViewModel> create(modelClass: Class<T>): T {
             return FormViewModel(
-                elements = formElements,
+                initialElements = formElements,
                 formArguments = formArguments,
             ) as T
         }
     }
 
+    private val linkInlineSignupMode = formArguments.linkHandler?.linkSignupMode?.stateIn(
+        scope = viewModelScope,
+        started = SharingStarted.Eagerly,
+        initialValue = null,
+    ) ?: stateFlowOf(null)
```

Comment:
The downside to creating and observing it here is that we will see `LinkElement` render in after the card details have rendered in since the initially posted value is `null`. Can be fixed if we properly update the upstream flows to be `StateFlow`.

Reply:
Is this a behavior change?